about.html

```diff
@@ -0,0 +1,150 @@
+       News
+       </li>
+    </a>
+            </ul>
+            <p class="popUpCancel">X</p>
+        </div>
+    </div>
+        <div class="hamburger">
+            <img src="icons8-hamburger-32.png" class="hamburgericon" id="hamburgericon" alt="hamburgericon"> 
+        </div>
+    </div>
+    <div class="navigationBar">
+        <img class="navigationLogo" src="mellonLogo.png">
+        <ul class="navigationItems flex">
+           <a href="#about"> <li class="navigationItem">
+                About
+            </li></a>
+           <a href="#program"> <li class="navigationItem">
+                Program
+            </li></a>
+            <a href="#join"> <li class="navigationItem">
+                Join
+            </li></a>
+           <a href="#sponsor">
+           <a href="#sponsor"> <li class="navigationItem">
+                Sponsor
+            </li>
+        </a>
+        </a>
+           <a href="#news"> <li class="navigationItem">
+                News
+            </li>
+        </a>
+        </ul>
+    </div>
+    <header class="headerAbout flex">
+        <h1 class="mainTitle">
+            "Mellon!""
+        </h1>
+        <h2 class="secondTitle">
+            ARE YOU READY  A JOURNEY TO  MIDDLE-EARTH?
+        </h2>
+    </header>
+    <main>
+    <div class="indexAbout flex" id="about">
+        <p class="description">
+            We meet every year on Tolkien's birthday date on 3 January and give our respect to his memory. It is a good opportunity to meet new people who are similar to you. Also, the legacy of Tolkien is lived by the way.  
+        </p>
+        <p class="indexDescription">
+            Please contact us per Email for any further questions about Mellor!</p>
+        <p class="email">
+burak.otman@bpdesign.com.tr
+        </p>
+    </div>
+    <div class="commonLogo flex">
+        <h2 class="logoTitle">
+            bp Organization <br> Mellor's Logo
+        </h2>
+        <p class="logoDescription">
+            The logo of Mellor was decided by coming from votes of Tolkien fans from 1. September to 1. November in 2022. 
+        </p>
+        <img src="./mellonLogo.png" class="mellorLogo" alt="mellorLogo">
+    </div>
+    <div class="pastEvents flex" id="news">
+<h2 class="pastEventsTitle">
+    See the past Mellon Events
+</h2>
+<p class="pastEventsDescription">
+    Take a look at last two Mellon Events which were in Simav and Zonguldak in Turkey.
+</p>
+<div class="eventImage flex">
+<div class="golcuk flex">
+    <div class="imageLayout">
+        <div class="eventIndex">
+    <h2 class="eventTitle">
+        2021
+    </h2>
+    <p class="eventDescription">
+        Mellon 2021 in Simav
+    </p>
+    </div>
+</div>
+</div>
+<div class="zonguldak flex">
+    <div class="imageLayout">
+        <div class="eventIndex">
+    <h2 class="eventTitle">
+        2022
+    </h2>
+    <p class="eventDescription">
+        Mellon 2022 in Zonguldak
+    </p>
+</div>
+</div>
+</div>
```

Comment:
- [ ] Kindly increase the size of the images. They are currently small and the text does not have enough space to match the Bhenace design

**This is how your design looks like right now**

![image](https://user-images.githubusercontent.com/86318284/212129571-603ef6eb-960a-4920-a005-4a5feb65991f.png)


**This is the desired design**

![image](https://user-images.githubusercontent.com/86318284/212129526-17230b99-3a83-43ce-86b8-d3c4da538027.png)